configs/default/AIRB-NOX.config

```diff
@@ -0,0 +1,93 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Oct 16 2019 / 11:57:34 (c37a7c91a) MSP API: 1.42
+
+board_name NOX
+manufacturer_id AIRB
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 A01
+resource MOTOR 2 A07
+resource MOTOR 3 B08
+resource MOTOR 4 B01
+resource PPM 1 B10
+resource LED_STRIP 1 A00
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 11 A02
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 11 A02
+resource INVERTER 2 C14
+resource LED 1 A04
+resource SPI_SCK 1 B03
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 B04
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 B05
+resource SPI_MOSI 2 B15
+resource ESCSERIAL 1 B10
+resource ADC_BATT 1 A05
+resource BARO_CS 1 A09
+resource FLASH_CS 1 A15
+resource OSD_CS 1 A10
+resource GYRO_EXTI 1 A08
+resource GYRO_CS 1 B12
+
+# timer
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+timer A01 AF2
+# pin A01: TIM5 CH2 (AF2)
+timer A07 AF1
+# pin A07: TIM1 CH1N (AF1)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+timer A02 AF3
+# pin A02: TIM9 CH1 (AF3)
+timer A03 AF3
+# pin A03: TIM9 CH2 (AF3)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+dma pin A07 0
+# pin A07: DMA2 Stream 6 Channel 0
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
```

Comment:
Should have `feature OSD` as well.

Reply:
To be consistent with the legacy target it should also have:
```
feature SOFTSERIAL
feature ESC_SENSOR
```